linkerd/app/outbound/src/http/endpoint.rs

```diff
@@ -5,6 +5,7 @@ use super::{
     NewRequireIdentity,
 };
 use crate::{tcp::tagged_transport, Outbound};
+use linkerd_app_core::metrics::OutboundZoneLocality;
 use linkerd_app_core::{
```

Comment:
nit: this should be placed with the other `linkerd_app_core` imports

Reply:
Done